### DISCUSSION TITLE
Free Code Signing

### DISCUSSION THREAD

**Discussion by ThomasNieto** [gerardog/gsudo#281] [Ideas]
Hello, I make use of this project quite a bit and noticed on your donation/support page that you're requesting assistance for the costs of a signing certificate.

I recently came across [0install](https://github.com/0install/0install-win#privacy-and-code-signing-policy) GitHub and noticed on their readme that they receive free code signing from [SignPath.io](https://about.signpath.io/product/open-source) for open source projects. I'm not affiliated with either project but thought it might reduce the upkeep cost for this awesome project.

**Comment by gerardog** [comment]
The donation page is somewhat outdated. Last year I received a free 3yr certificate [donation from Parag Mehta](https://twitter.com/gerardo_gr/status/1595817336405168134) from https://signmycode.com/. So, two more years to go until I need a new one! I am using that certificate in my automated gsudo build pipelines and works great.
Once I'm on the last year, I'll consider SignPath.io too. Thanks for letting me know about it!
